cms/static/sass/bootstrap/_variables.scss

```diff
@@ -57,7 +57,7 @@ $footer-bg: $white !default;
 $font-light: 300 !default;
 $font-regular: 400 !default;
 $font-semibold: 600 !default;
-$font-bold: 700 !default;
+$font-bold: 600 !default;
```

Comment:
Should we make semibold 500 to keep them different?

Reply:
Good catch. I don't know how that change snuck in, as I'm not working on Studio. I'll revert it.

Reply:
@andy-armstrong the CSS keyword ```bold``` is ```700```, why move our bold variable to ```600```? I fear it may cause confusion.

Reply:
Agreed @AlasdairSwan. I don't know how that change got in (it must have been during the heat of the hackathon) but I've reverted it now.